gerrit/src/test/java/com/ruesga/rview/gerrit/filter/ChangeQueryTest.java

```diff
@@ -97,17 +97,21 @@ public void testParseQuery() {
                                 .and(new ChangeQuery().branch("test2"))));
 
 
-        testParseQuery("(is:open label:Verified=+1 -age:1month) AND NOT (is:open reviewerin:Tuleap-Integrators -age:1month label:Verified=+1 -is:draft)",
+        ChangeQuery complexQuery = new ChangeQuery();
+        complexQuery.add("(" +
```

Comment:
"add" is an internal method and is not allowed to be called by others but the package classes. Instead, I rather add a new wrap function to handle the described issue.